NEWS.md

```diff
@@ -67,6 +67,9 @@
 
 * `build_home()`: a link to the source `inst/CITATION` was added to the authors page (#714).
 
+* `prepend_class()`: Generalized to allow `tweak_tables()` to handle multiple
```

Comment:
`prepend_class()` is not exported so this is confusing to a reader.

Can you make it:

```
* Tables with multiple attributes are now properly handled (@yonicd #868).
```